docs/paper/admin/how-to/basic-troubleshooting.md

```diff
@@ -98,6 +98,15 @@ To efficiently search for the plugin that is causing the issue, you can do the f
 4. **Repeat from the start with the suspect plugin group**
    Repeat the steps above with groups that have the plugin that is causing the issue.
 
+:::tip
```

Comment:
should probably be a warning to emphasize that they _will_ break their server by removing library plugins